controller.go

```diff
@@ -426,6 +435,48 @@ func (c *Controller) syncHandler(key string) error {
 			}
 
 			id = res.ID
+
+		} else if c.infraConfig.Provider == "gce" {
+			provisioner, _ := provision.NewGCEProvisioner(ctx, c.infraConfig.GetAccessKey())
+			userData := makeUserdata(tunnel.Spec.AuthToken, c.infraConfig.UsePro(), tunnel.Spec.ServiceName)
+			firewallPort := strconv.Itoa(inletsControlPort)
+			if c.infraConfig.UsePro() {
+				firewallPort = strconv.Itoa(inletsProControlPort)
+			}
+			firewallRuleName := "inlets"
+			exists, err := provisioner.CheckFirewallRuleExists(c.infraConfig.ProjectID, firewallRuleName, firewallPort)
+			log.Printf("FirewallRuleExists: %v", exists)
+
+			if err != nil {
+				return err
+			}
+			if !exists {
+				_, err := provisioner.CreateInletsFirewallRule(c.infraConfig.ProjectID, firewallRuleName, firewallPort)
+				if err != nil {
+					log.Printf("Could not create firewallRule for port: %v and targetTag: %v. Create FirewallRule manually!", firewallPort, firewallRuleName)
+				} else {
+					log.Printf("FirewallRule created for inletsControlPort %s", firewallPort)
+				}
+			}
+
+			res, err := provisioner.Provision(provision.BasicHost{
+				Name:     tunnel.Name,
+				OS:       "projects/debian-cloud/global/images/debian-9-stretch-v20191014",
```

Comment:
What was the reason for not using Ubuntu, so that I know for future reference?

Reply:
The first reason was that I wanted to know if we can run inlets on Debian considering the rest of the providers are using Ubuntu. Also although I have not verified it but considering it is lightweight, I guess it boots up quicker than Ubuntu?